repos/jsutils/package.json

```diff
@@ -15,7 +15,8 @@
   },
   "homepage": "https://github.com/simpleviewinc/keg-hub#readme",
   "files": [
-    "build"
+    "build",
+    "src/node"
```

Comment:
It's so funny cause I was just thinking how we needed to do something like this.

I was planning to add a hook, to copy the node folder into the build folder, but this is so much better